<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-04JSCC6" name="Test1" tests="8" failures="3" timestamp="18 Jul 2019 05:37:56 GMT" time="178.176" errors="0">
  <testcase name="TC004_SampleScript_Test" time="9.351" classname="datadriven.testscript.TC004_SampleScript">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at datadriven.testscript.TC004_SampleScript.TC004_SampleScript_Test(TC004_SampleScript.java:254)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- TC004_SampleScript_Test -->
  <testcase name="ValidationTest" time="13.421" classname="datadriven.testscript.TC003_SampleScript"/>
  <testcase name="TC004_SampleScript_Test" time="0.002" classname="datadriven.testscript.TC004_SampleScript">
    <skipped/>
  </testcase> <!-- TC004_SampleScript_Test -->
  <testcase name="TC004_SampleScript_Test" time="38.302" classname="datadriven.testscript.TC004_SampleScript">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at datadriven.testscript.TC004_SampleScript.TC004_SampleScript_Test(TC004_SampleScript.java:133)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- TC004_SampleScript_Test -->
  <testcase name="ValidationTest" time="61.801" classname="datadriven.testscript.TC003_SampleScript"/>
  <testcase name="TC004_SampleScript_Test" time="40.575" classname="datadriven.testscript.TC004_SampleScript">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at datadriven.testscript.TC004_SampleScript.TC004_SampleScript_Test(TC004_SampleScript.java:133)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- TC004_SampleScript_Test -->
  <testcase name="ValidationTest" time="12.936" classname="datadriven.testscript.TC003_SampleScript"/>
  <testcase name="ValidationTest" time="13.634" classname="datadriven.testscript.TC003_SampleScript"/>
</testsuite> <!-- Test1 -->
